source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,57 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
```

Comment:
Nitpick: comma is unnecessary here since what follows it is not an independent clause.

Reply:
I think it's actually valid the way it's currently written. See: https://www.grammarly.com/blog/such-as-comma/

![Screen Shot 2020-02-03 at 5 14 34 PM](https://user-images.githubusercontent.com/52428905/73695736-b4345500-46a8-11ea-9a4a-ce9cfb31ea39.png)